### PR TITLE
Fixes #22702: When we clone a group in Rudder 7.2, newly added properties are not cloned

### PR DESCRIPTION
**Pull request by ElaadF**
https://issues.rudder.io/issues/22702

The idea is to get the NodeGroup info refreshed, because when we clone we doesn't get back modifications that can be made without "Save" button up right
Property management is done by an angular app shared between node property and node group

### CODE REVIEW

File: webapp/sources/rudder/rudder-web/src/main/webapp/javascript/rudder/rudder.js [hunk 1]
```diff
@@ -418,6 +418,11 @@ function makeDiff(beforeId,afterId,resultId) {
     fragment.appendChild(node);
   }
 
+  result.css({
+    "white-space": "pre-line",
+    "word-break": "break-word",
+    "overflow": "auto"
+  });
```
Comment (ElaadF) [inline webapp/sources/rudder/rudder-web/src/main/webapp/javascript/rudder/rudder.js:425]: Should remove this changes, it's a commit from another branch

**Comment by ElaadF** [comment]
PR updated with a new commit

**Comment by ElaadF** [comment]
PR rebased

**Review by fanf — APPROVED** [review]
LGTM

**Pull request merged**